doc/changes/changes_next.md

```diff
@@ -0,0 +1,7 @@
+# Virtual Schema Common Java X.X.X, released 2020-11-12
```

Comment:
are you planning to leave the version open until a next release?

Also, the entry in the changelog file is missing as well

Reply:
and the date is wrong

Reply:
```suggestion
# Virtual Schema Common Java X.X.X, released 2020-XX-XX
```

Reply:
Yes. Since I don't know weatcher the next release will be a fix, feature, or API change, I don't know the next version.
Therefore I also can't link the file from the changelog.md since it's name will change